db/seeds.rb

```diff
@@ -3,3 +3,47 @@
 # with its default values.
 # The data can then be loaded with the rails db:seed command (or created
 # alongside the database with db:setup).
+
+account = Account.first!
```

Comment:
Part of what I'd really like to see with this work is prod-like data, meaning many accounts with a representative set of policies, each with some largish number of hosts assigned (like 2 to 1000 hosts per policy) and with at least some significant portion of hosts reporting across different SSG versions with different levels of supported/unsupported and compliant/non-compliant.

Reply:
Yeah, I was trying to approach this from the UI perspective, so generate stuff that the user can navigate through. But your perspective also makes sense, I guess we can do both...